effects/butterfly-gears/butterfly-gears.c

```diff
@@ -0,0 +1,167 @@
+#include "startup.h"
+
+#include "hardware.h"
+#include "interrupts.h"
+#include "blitter.h"
+#include "color.h"
+#include "copper.h"
+#include "bitmap.h"
+#include "palette.h"
+#include "pixmap.h"
+#include "memory.h"
+#include "sprite.h"
+#include "fx.h"
+
+#define ROTZOOM_W 24
+#define ROTZOOM_H 24
+#define COPWAIT_X 1
+#define Y0 Y((256-280)/2)
+#define COPPER_HALFROW_INSTRUCTIONS ROTZOOM_W/2+2
```

Comment:
Please use parenthesis around expressions in such cases. I've run into problems on several occasions when macrodefinition was instantiated.

Reply:
fixed